projects/packages/my-jetpack/src/class-products.php

```diff
@@ -156,7 +209,7 @@ public static function get_product_data_schema() {
 				'status'      => array(
 					'title' => 'The product status',
 					'type'  => 'string',
-					'enum'  => array( 'active', 'inactive', 'plugin_absent', 'needs_purchase', 'needs_purchase_or_free', 'needs_first_site_connection', 'user_connection_error', 'site_connection_error' ),
+					'enum'  => array( 'active', 'inactive', 'plugin_absent', 'plugin_absent_with_plan', 'needs_purchase', 'needs_purchase_or_free', 'needs_first_site_connection', 'user_connection_error', 'site_connection_error', 'can_upgrade', 'module_disabled' ),
```

Comment:
Considering that we now have the consts for each status, what do you think of creating an `$all_module_statuses` (or something like) with all status available and use them here in the `enum`/

Reply:
That's a nice suggestion, it's easy so I'll go ahead and do it now 😄